lms/templates/learner_dashboard/program_card.underscore

```diff
@@ -0,0 +1,24 @@
+
+<div class="banner-image">
+    <a href="<%- marketingUrl %>" class="card-link">
+        <img alt-text="<%- gettext(name)%>" src="" />
+    </a>
+</div>
+<div class="text-section">
+    <div class="meta-info">
+        <div class="organization" aria-hidden="true">
```

Comment:
I thought this is what we have agreed with Mark, that the `organization` and `category` would crowd the screenreader so the `name`, `organization` and `category` would be mumbo together. I thought hide these values from screenreader would make the cards easier to understand. Am I interpreting this wrong?